### PR TITLE
Add EIP: Bitcoin Proof-of-Stake

### PR DESCRIPTION
**Pull request by etan-status**

### CODE REVIEW

**Comment by wjmelements** [comment]
Shouldn't this be a BIP?

**Comment by emmanuel-awosika** [comment]
> Shouldn't this be a BIP?

Since the proposal requires buy-in from the Ethereum community (mainly node operators), it makes sense to put out an EIP first. The current spec does include a phase where a BIP is created to enable the transition, though. Relevant quote from the `Stage 3: Signalling in Bitcoin blocks` section: 

> When >= 30% of produced Ethereum blocks over a sliding 4-week period (`6*24*7*28` = 28'224 blocks) indicate support for this EIP by including a valid Bitcoin block hash in their graffiti, a Bitcoin Improvement Proposal (BIP) SHALL be created defining a version bit that represents support by miners to eventually transition to Ethereum Proof-of-Stake.

@wjmelements

File: EIPS/eip-####.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
```
Comment (abcoathup) [inline EIPS/eip-####.md:1]:
```suggestion
---
eip: 7671
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.

File: EIPS/eip-####.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+title: Bitcoin Proof-of-Stake
+description: Bitcoin transition from Proof-of-Work to Proof-of-Stake
+author: Etan Kissling (@etan-status)
+discussions-to: https://ethereum-magicians.org/t/eip-bitcoin-proof-of-stake/19477
```
Comment (abcoathup) [inline EIPS/eip-####.md:5]:
```suggestion
discussions-to: https://ethereum-magicians.org/t/eip-7671-bitcoin-proof-of-stake/19477
```

**Comment by etan-status** [comment]
- Renamed document to reflect assigned EIP number
- Added Rationale, Backwards Compatibility and Security Considerations sections
- Fixed typos and incorrect computations for number of blocks, adjusted period from from 12-weeks to 13-weeks (quarter year)

**Comment by SamWilsn** [comment]
\*checks date\* Let me know if you intend to pursue this.

**Pull request closed without merging**